src/main/java/edu/oakland/production/display01/DisplayGpsInterfaceImplementation.java

```diff
@@ -1,23 +1,67 @@
 package edu.oakland.production.display01;
 
 import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsReceiver;
 
 public class DisplayGpsInterfaceImplementation implements DisplayGpsInterface {
+  
+  public DisplayGpsReceiver reciever;
 
-  public boolean receiveGpsSignal(Satellite satelliteSignal) {
-    return true;
+  public DisplayGpsInterfaceImplementation(DisplayGpsReceiver reciever) {
+    this.reciever = reciever;
```

Comment:
Can you add a null checker? Any other class that is merged to main should be able to be used as an example. There is a test associated with this class as well.